lxqt-admin-user/lxqt-admin-user-helper

```diff
@@ -2,10 +2,21 @@
 case "$1" in
 # we only allow executing these commands
 useradd|usermod|userdel|groupadd|groupmod|groupdel|passwd|gpasswd)
-    # TODO: platforms using different commands can add wrapper scripts here.
     export LC_ALL=C
     exec "$@"
     ;;
+pw)
```

Comment:
Looking at this again... wouldn't it be better to provide a BSD specific script?

I mean leaving the  `lxqt-admin-user-helper` as is in the sources (or rename it to `lxqt-admin-user-helper.default`) and create new one `lxqt-admin-user-helper.bsd` just with the BSD specific things. In build time, the correct one will be chosen and installed as `lxqt-admin-user-helper`. The installed run-time script won't be cluttered with irrelevant stuff from other platforms

What do you think?